overlord/snapstate/backend_test.go

```diff
@@ -1174,6 +1195,14 @@ func (f *fakeSnappyBackend) RemoveSnapCommonData(info *snap.Info, opts *dirs.Sna
 	return f.maybeErrForLastOp()
 }
 
+func (f *fakeSnappyBackend) RemoveSnapSaveData(info *snap.Info) error {
+	f.appendOp(&fakeOp{
+		op:   "remove-snap-save-data",
+		path: info.MountDir(),
```

Comment:
any reason not to capture CommonDataSaveDir here?

Reply:
That was definitely an oversight, thanks for catching this!